src/imp/noop_imp.js

```diff
@@ -1,18 +1,25 @@
+function NoopSpanContextImp() {
+}
+NoopSpanContextImp.prototype.setBaggageItem = function (key, value) {
+}
+NoopSpanContextImp.prototype.getBaggageItem = function (key) {
+}
+
 function NoopSpanImp() {
 }
 NoopSpanImp.prototype.tracer = function () {
     return null;
 }
+NoopSpanImp.prototype.context = function () {
+    // XXX what's the deal here? Why can tracer() return null?
```

Comment:
why wouldn't `NoopSpanImp` be initialized with `NoopTracer` and `NoopContext`? I would prefer never to return null from any methods.